geoprocessing-tools.qmd

```diff
@@ -34,21 +34,18 @@ determined by the **direction** of a parameter.
 
 ![](images/gp-tool-properties.png){width=50%}
 
-If the direction is `Input` it will be contained in `in_params`. Likewise, 
+If the direction is `Input`, it will be contained in `in_params`. Likewise, 
 if the direction is `Output`, it will be contained in `out_params`. 
 
 ## Using parameters
 
-We can extract values from the `in_params` and `out_params` lists by the name
-of the parameter. 
+Values can be extracted from the `in_params` and `out_params` lists by the name or position of the parameter. It is highly recommended to use name-based indexing rather than position-based indexing for clarity.
```

Comment:
```suggestion
Values can be extracted from the `in_params` and `out_params` lists by the name or position of the parameter. It is strongly recommended to use name-based indexing rather than position-based indexing for clarity.
```